.github/workflows/build.yml

```diff
@@ -2,7 +2,7 @@ name: Build
 
 on:
   push:
-    branches: [main]
+    # branches: [main]
```

Comment:
to uncomment !

Reply:
I think we can keep the build for any push on any branch, like in Station